Makefile.am

```diff
@@ -102,6 +103,11 @@ ppmclibs/Makefile: ppmclibs/Makefile.PL
 $(PERL_MODULE): ppmclibs/Makefile
 	$(MAKE) -C ppmclibs
 
+$(OPENVSWITCH_SERVICE):
+	rm -f $@ $@.tmp
+	sed -e 's,@pkglibexecdir[@],$(pkglibexecdir),g' '$(top_srcdir)/$@.in' > '$(top_srcdir)/$@.tmp'
```

Comment:
right, I overlooked. But this replacement of @pkglibexecdir@ is done by autoconf if you list it in configure.ac